app/controllers/reads_controller.rb

```diff
@@ -1,10 +1,13 @@
 class ReadsController < ApplicationController
+
   def create
-    current_user.reads.create!(text_book_id: params[:text_book_id])
-    redirect_back(fallback_location: root_path)
+    @text_book = TextBook.find(params[:text_book_id])
+    current_user.reads.create!(text_book_id: @text_book.id).save
```

Comment:
`create!` は `new` + `save!` の意味ですので，`save` は不要です。

```suggestion
    current_user.reads.create!(text_book_id: @text_book.id)
```

Reply:
訂正いたしました。